.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,17 @@
 To help us process your pull request efficiently, please replace the 
 comments shown below. 
 
+## A Pull Request should be associated with an Issue.
+
+We wish to have discussions in Issues. A single issue may be targeted by multiple PRs.
+If you're offering a new feature or fixing anything, we'd like to know beforehand in Issues,
+and potentially we'll be able to point development in a particular direction.
+
+Related issue: https://github.com/vmware/go-vcloud-director/issues/0123456789
+
+If the PR is a really trivial change (such as a spelling correction) then go ahead and
```

Comment:
Suggested change:  We accept PRs without associated issues provided the change is sufficiently evident from the commit message.  If you have typos or simple bug fixes go for it.